encoding.bs

```diff
@@ -1351,13 +1357,17 @@ TextDecoder includes TextDecoderCommon;
 initially false.
 
 <dl class=domintro>
+ <dt>{{TextDecoder}} . <a method for=TextDecoder lt=decode()>decode([<var>input</var> [, <var>options</var>]])</a></code>
+ <dd>
+  <p>Returns the result of running <a for=TextDecoderCommon>encoding</a>'s <a for=/>decoder</a>.
```

Comment:
Below this should probably be prefixed with _decoder_'s and here it needs some rewording as there is no instance to speak of.